app/assets/stylesheets/components/_modal.scss

```diff
@@ -46,53 +46,58 @@ $modal-padding-laptop: $modal-button-laptop;
     @include breakpoint($medium) {  padding-top: $modal-padding-laptop; }
   }
 
-    &__close {
-      @include modal-button-sizing($modal-button-mobile, $modal-button-tablet, $modal-button-laptop);
-      background-color: $grey-dark;
-      border-radius: 100%;
-      cursor: pointer;
-    
-      position: absolute;
-
-      &::after {
-        @extend .icon--cross;
-        content: '';
-        margin: auto;
+  &__close {
+    @include modal-button-sizing($modal-button-mobile, $modal-button-tablet, $modal-button-laptop);
+    background-color: $grey-dark;
+    border-radius: 100%;
+    cursor: pointer;
+  
+    position: absolute;
 
-        display: block;
-      }
+    &::after {
+      @extend .icon--cross;
```

Comment:
To be honest, looking at the way it's been set up, I would ask that we overlook this for the time being.

All of the project's icons are basically written in classes, and overhauling it without breaking anything seems like it may take a fair amount of time.

If you agree, I would prefer to leave this for the time being and create an issue that can be looked at after this release so that we don't delay deploying the site any longer - what do you think?

<img width="636" alt="image" src="https://user-images.githubusercontent.com/56026099/126334089-09e3b687-047c-4a40-bcfd-11df69a36501.png">